src/js/components/DataTable/DataTable.js

```diff
@@ -222,7 +222,7 @@ const DataTable = ({
       }
       const nextPinnedOffset = {};
 
-      if (columnWidths !== []) {
+      if (columnWidths.length !== 0) {
```

Comment:
Tiny question, is this same as `if (columnWidths?.length)`

Reply:
Given
```
const columnWidths = [];
```

```
if (columnWidths?.length)
```
would evaluate to `0` which would be falsey and not `false`, right? Thinking I'd prefer the explicit. Thoughts?

Reply:
Good with the more explicit.